### ISSUE TITLE
AI powered customized chatbot contents

### ISSUE THREAD

**Issue by viuts** [zeals-co-ltd/technical_debts#3]
# About

We believe that empowering AI to finetune the conversation for each end user will help to improve end user experience, but due to the limitation of data and improper user persona filing, we are unable to develop a generic machine learning model for that.

Also we already for each industry, there are also different approaches and contexts during the conversation, an AI-powered model should be also able to handle that as well.

# Actions

1. Massage / Propose a data structure that works for the longer run
2. Research on NLP/NLG
3. Fine-tuning the model to balance between accuracy and speed/size

# Positions

- [ML engineer](https://github.com/zeals-co-ltd/technical_debts/issues/4)
- [Backend engineer](https://github.com/zeals-co-ltd/technical_debts/issues/5)
